developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -47,35 +47,72 @@ const baseOptions: CompilerOptions = {
  */
 let callbackProcIdentifier = 0;
 
+/**
+ * The wrapped functions
+ */
+class WrappedWasmFuncs {
+  compileKeyboardFile?: (a0: string, a1: string, a2: number, a3: number, a4: number, a5: string) => boolean;
+  parseUnicodeSet?: (a0: string, a1: number, a2: number) => number;
+  setCompilerOptions?: (shouldAddCompilerVersion: number) => boolean;
+
+  constructor(wasmModule: any) {
+    this.compileKeyboardFile = wasmModule.cwrap('kmcmp_Wasm_CompileKeyboardFile', 'boolean', ['string', 'string', 'number', 'number', 'number', 'string']);
+    this.parseUnicodeSet = wasmModule.cwrap('kmcmp_Wasm_ParseUnicodeSet', 'number', ['string', 'number', 'number']);
+    this.setCompilerOptions = wasmModule.cwrap('kmcmp_Wasm_SetCompilerOptions', 'boolean', ['number']);
+  }
```

Comment:
I think we should consider a `throw` here if this fails as it's a fatal error, and then we can eliminate `ok()`?